package.json

```diff
@@ -579,6 +584,10 @@
 			  "command": "bitbake.start-toaster-in-browser",
 			  "group": "1@bitbake_dev@4"
 			},
+			{
+				"command": "bitbake.examine-dependency-taskexp",
```

Comment:
You should also add this in view/item/context". Basically copy where "bitbake.recipe-devshell" is referenced.

Reply:
done